src/Util/EMailer/SystemMailBuilder.php

```diff
@@ -98,7 +94,7 @@ protected function getSubject()
 	 */
 	protected function getHtmlMessage()
 	{
-		$htmlVersion = BBCode::convert($this->body);
+		$htmlVersion = !empty($this->body) ? BBCode::convert($this->body) : '';
```

Comment:
Please default the `$body` parameter to empty string on line 72 to avoid this check.